File: billion.py
import time

def count_to_billion():
    start_time = time.time()

    for i in range(1, 1000000001):
        pass  # The loop does nothing but iterate

    end_time = time.time()
    duration = end_time - start_time
    print(f"Counting from 1 to 1,000,000,000 took {duration:.2f} seconds.")


if __name__ == "__main__":
    count_to_billion()
  

File: billion.cpp
#include <iostream>
#include <chrono>

int main() {
    auto start = std::chrono::high_resolution_clock::now();

    for (long long i = 1; i <= 1000000000; ++i) {
        // The loop does nothing but iterate
    }

    auto end = std::chrono::high_resolution_clock::now();
    std::chrono::duration<double> duration = end - start;

    std::cout << "Counting from 1 to 1,000,000,000 took " << duration.count() << " seconds." << std::endl;

    return 0;
}
